### DISCUSSION TITLE
Allow users to know how big zip downloads will be

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#48234] [Repositories]
### Select Topic Area

Product Feedback

### Body

Here we are, about to press the Download Zip button.
![xx](https://user-images.githubusercontent.com/1230959/220526535-aaf072cf-64d6-4ba4-ab6f-25d1cbc5898e.jpg)
Are we going to download a kilobyte, megabyte, gigabyte?
Please let the user know by showing an approximate size net to the button!

**Comment by ghost** [comment]
Hi.

You are super correct. For example, I never know the size of the repository or the size of the zip file. In this example you passed there, it's my case. I never know the size of the repository or downloaded file. I would be happy if GitHub implemented the zip file size feature, so many users could benefit from a feature.
